domain/src/main/java/org/mobilitydata/gtfsvalidator/domain/entity/stoptimes/TimepointStoptime.java

```diff
@@ -0,0 +1,37 @@
+package org.mobilitydata.gtfsvalidator.domain.entity.stoptimes;
+
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+public class TimepointStoptime extends TimepointBase {
+
+    public TimepointStoptime(@NotNull String tripId,
+                             @NotNull String arrivalTime,
+                             @NotNull String departureTime,
+                             @NotNull String stopId,
+                             int stopSequence,
+                             @Nullable String stopHeadsign,
+                             @NotNull PickupType pickupType,
+                             @NotNull DropOffType dropOffType,
+                             @Nullable Float shapeDistTraveled) {
+        super(tripId, arrivalTime, departureTime, stopId, stopSequence, stopHeadsign, pickupType, dropOffType,
+                shapeDistTraveled);
+    }
+
+    public static class TimepointStoptimeBuilder extends TimepointBaseBuilder {
+
+        public TimepointStoptimeBuilder(@NotNull String tripId,
+                                        @NotNull String arrivalTime,
+                                        @NotNull String departureTime,
+                                        @NotNull String stopId,
+                                        int stopSequence) {
+            super(tripId, arrivalTime, departureTime, stopId, stopSequence);
+
+        }
+
+        public TimepointStoptime build() {
+            return new TimepointStoptime(tripId, arrivalTime, departureTime, stopId, stopSequence, stopHeadsign,
```

Comment:
Could base class constructor be used?

Reply:
See https://github.com/MobilityData/gtfs-validator/pull/88#discussion_r399400056

Reply:
Last decision about the architecture of this class avoids this problem: one unique class StopTime representing all stop_time objects.